Dumper/TranslationDumper.php

```diff
@@ -194,6 +211,41 @@ private function dumpTranslations($route, array $formats, $target)
         }
     }
 
+    private function dumpMergedTranslations($route, array $formats, $target)
+    {
+        foreach ($this->getTranslations() as $locale => $domains) {
+            foreach ($formats as $format) {
+                $content = $this->engine->render(
+                    'BazingaJsTranslationBundle::getTranslations.' . $format . '.twig',
+                    array(
+                        'translations' => array($locale => $domains),
+                        'include_config' => false,
+                    )
+                );
+
+                $file = sprintf(
+                    '%s/%s',
+                    $target,
+                    strtr(
+                        $route->getPath(),
+                        array(
+                            '{domain}' => sprintf('%s/%s', $locale, $locale),
```

Comment:
well, I don't really agree here (and some people might prefer grouping all JS files than grouping all english files btw)